### PR TITLE
chore: add fakeai provider

### PR DESCRIPTION
**Pull request by thschue**
Closes # 

## 📑 Description
* Create an fakeai provider for local development, testing and troubleshooting
* This provider simply echoes the input given as problem

## ✅ Checks

- [X] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [X] I have updated the documentation as required
- [X] All the tests have passed

## Additional Information
The output looks like this: 
image

### CODE REVIEW

File: pkg/ai/ai.go [hunk 1]
```diff
@@ -87,3 +87,7 @@ func (a *OpenAIClient) Parse(ctx context.Context, prompt []string, nocache bool)
 	}
 	return response, nil
 }
+
+func (a *OpenAIClient) GetName() string {
+	return "fakeai"
```
Comment (matthisholleville) [inline pkg/ai/ai.go:92]: Shouldn't it be `openai` instead of `fakeai` here?

**Comment by matthisholleville** [comment]
Shouldn't we rename `ai.go` to `openai.go` now that we have two backends? Anyway, this fake client can easily help us develop more comprehensive tests on the core features of the tool. Well done!

File: pkg/ai/ai.go (hunk 1 above)
Comment (AlexsJones) [reply pkg/ai/ai.go:92]: +1 on this

File: pkg/ai/fakeai.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+import (
+	"context"
+	"encoding/base64"
+	"github.com/fatih/color"
+	"github.com/spf13/viper"
+	"strings"
+)
+
+type FakeAIClient struct {
```
Comment (AlexsJones) [inline pkg/ai/fakeai.go:11]: Another name could be `NoOpAIClient`

File: pkg/ai/ai.go (hunk 1 above)
Comment (thschue) [reply pkg/ai/ai.go]: fixed this

File: pkg/ai/fakeai.go (hunk 2 above)
Comment (thschue) [reply pkg/ai/fakeai.go]: renamed it

**Comment by thschue** [comment]
> Shouldn't we rename `ai.go` to `openai.go` now that we have two backends? Anyway, this fake client can easily help us develop more comprehensive tests on the core features of the tool. Good job!

Renamed it to openai.go

**Comment by matthisholleville** [comment]
Regarding the latest minor detail, you indicated that the modification made required an update in the documentation, which has already been modified. However, I have noticed that the readme file is missing in this commit. Could you confirm if this is an expected outcome?

**Comment by thschue** [comment]
> Regarding the latest minor detail, you indicated that the modification made required an update in the documentation, which has already been modified. However, I have noticed that the readme file is missing in this commit. Could you confirm if this is an expected outcome?

Created an Issue here: https://github.com/k8sgpt-ai/docs/issues/13

**Review by AlexsJones — APPROVED** [review]

**Review by AlexsJones — APPROVED** [review]

**Pull request merged**